src/db-schema.jl

```diff
@@ -246,68 +263,55 @@ function create_views(db::SQLite.DB)
         """,
 
         """
-        -- Table to summarize what posts a user is/is not informed of (has/has not voted on)
-        -- This table implements two important rules.
-        -- 1. A user has considered a post if they have considered its parent.
-        -- 2. Only count user as uninformed of a post when the user is informed of the parent of the post.
-        create view InformationStatus as
-        with informed as ( 
-          -- first, find all posts that user is informed of (where they have voted on that post OR a parent)
-          select user_id, tag_id, post_id, 2 as event_type 
-          from vote
-          where vote != 0
-          -- and parent_id is null
-          UNION 
-          select user_id, tag_id, ancestor_id as post_id, 2 as event_type
-          from lineage
-          join vote on
-            descendant_id = vote.post_id
-            -- and parent_id is null
-          where vote != 0
+        create view ImplicitlyInformedVote as 
+          select
+            iv.user_id
+            , iv.tag_id
+            , iv.post_id
+            , lineage.ancestor_id as note_id
+            , iv.vote as vote
+            , (iv.informed or ifnull(vote.vote,0) != 0) as informed
+            , iv.note_id as informed_by_post_id
+          from 
+          InformedVote iv
+          join lineage
+            on lineage.descendant_id = iv.note_id
+            and lineage.ancestor_id > iv.post_id
+          left join vote 
+            on vote.post_id = lineage.ancestor_id
+            and vote.tag_id = iv.tag_id
+            and vote.user_id = iv.user_id
+        ;     
+        """,
+
+        """
+        create view ConditionalTally as 
+        with overall as (
+          select * from InformedTally
+          UNION ALL
+          select tag_id, post_id, post_id, count, total
+          from tally
```

Comment:
Is a tally for (postId, postId) your convention to denote uninformed tallies?

Reply:
Kind of. The uninformed tally might be better understood as the "partially informed tally" -- it is the tally of votes for the target among users who are informed of the parent of the note, but not the note itself. I talk about this a little here: https://github.com/social-protocols/internal-wiki/blob/main/pages/research-notes/2024-05-24-calculating-tallies.md#detail-uninformed-tally-includes-users-who-are-informed-of-parent

So the partially informed tally for note C and target A is the informed tally of the parent of C and target A.

Now, if the note is a direct reply to the target, then the the parent of the note is the target. Thus the partially informed tally is the tally of votes for the target among users who are informed of the target -- which is just the overall tally for the target.